force-app/main/default/flows/Before_Save_Session_Registration_Validations.flow-meta.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Flow xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>54.0</apiVersion>
+    <assignments>
+        <name>Assign_JuncitonID_0</name>
+        <label>Assign JuncitonID</label>
+        <locationX>50</locationX>
+        <locationY>1463</locationY>
```

Comment:
Because session and attendee can not be changed, junction id creation and overlap check is not needed on update operation @bmtemakina